src/core/tests/Test_StatusHandler.py

```diff
@@ -401,183 +399,5 @@ def test_sort_packages_by_classification_and_state(self):
-        self.assertEqual(formatted_message['patches'][0]['name'], 'python-samba0')
-        self.assertEqual(formatted_message['patches'][0]['patchId'], expected_patch_id)
-
-    def test_installation_packages_map(self):
-        patch_id_other = 'python-samba0_2:4.4.5+dfsg-2ubuntu5.4_Ubuntu_16.04'
-        expected_value_other = {'version': '2:4.4.5+dfsg-2ubuntu5.4', 'classifications': ['Other'], 'name': 'python-samba0',
-                          'patchId': 'python-samba0_2:4.4.5+dfsg-2ubuntu5.4_Ubuntu_16.04', 'patchInstallationState': 'Installed'}
-
-        patch_id_critical = 'python-samba0_2:4.4.5+dfsg-2ubuntu5.4_Ubuntu_16.04'
-        expected_value_critical = {'version': '2:4.4.5+dfsg-2ubuntu5.4', 'classifications': ['Critical'], 'name': 'python-samba0',
-                          'patchId': 'python-samba0_2:4.4.5+dfsg-2ubuntu5.4_Ubuntu_16.04', 'patchInstallationState': 'Installed'}
-        self.runtime.execution_config.operation = Constants.INSTALLATION
-        self.runtime.status_handler.set_current_operation(Constants.INSTALLATION)
-
-        patch_count_for_test = 50
-        test_packages, test_package_versions = self.__set_up_packages_func(patch_count_for_test)
-
-        self.runtime.status_handler.set_package_install_status(test_packages, test_package_versions, 'Installed', 'Other')
-        with self.runtime.env_layer.file_system.open(self.runtime.execution_config.status_file_path, 'r') as file_handle:
-            substatus_file_data = json.load(file_handle)[0]["status"]["substatus"][0]
-
-        formatted_message = json.loads(substatus_file_data['formattedMessage']['message'])
-        self.assertEqual(len(formatted_message['patches']), patch_count_for_test)
-        self.assertEqual(formatted_message['patches'][0]['classifications'], ['Other'])
-        self.assertEqual(formatted_message['patches'][0]['patchId'], patch_id_other)
-        self.assertEqual(formatted_message['patches'][0], expected_value_other)
-        self.assertEqual(formatted_message['patches'][0]['name'], 'python-samba0')
-
-        # Update the classification from Other to Critical
-        self.runtime.status_handler.set_package_install_status_classification(test_packages, test_package_versions, 'Critical')
-        with self.runtime.env_layer.file_system.open(self.runtime.execution_config.status_file_path, 'r') as file_handle:
-            substatus_file_data = json.load(file_handle)[0]["status"]["substatus"][0]
-
-        formatted_message = json.loads(substatus_file_data['formattedMessage']['message'])
-        self.assertEqual(len(formatted_message['patches']), patch_count_for_test)
-        self.assertEqual(formatted_message['patches'][0]['classifications'], ['Critical'])
-        self.assertEqual(formatted_message['patches'][0]['patchId'], patch_id_critical)
-        self.assertEqual(formatted_message['patches'][0], expected_value_critical)
-        self.assertEqual(formatted_message['patches'][0]['name'], 'python-samba0')
```

Comment:
test_installation_packages_map() and test_assessment_packages_map() are not changes brought about by truncation. They should be kept here

Reply:
do we rlly need to move these functions back to Test_StatusHandler.py because these functions are using the same setup functions below? if I move them back then Test_StatusHandler.py will have the same set up functions, that's unnecessary code duplication, this is why i have them in the Test_StatusHandlerTruncation.py and they are created related to truncations in some way. 
__set_up_packages_func, __read_tmp_log_and_assert, __remove_tmp_file_reset_stdout, __create_tmp_file_for_log_and_set_stdout

Reply:
pending

Reply:
Test_StatusHandlerTruncation.py, as it's name suggests, is something that should contain everything related to truncation of the status file. It should not contain things, that it is not designed for, that is code leak.

These mapping tests were added before any truncation code, how were these designed back then? Why are they changed in this PR, where we aren't making any changes to the mapping logic?

Reply:
I see a lot of other functions from here also removed, which are not testing the truncation logic at all. They are pre-existing from before any of the truncation changes.

Re-iterating, the ask for a separate test file for truncation, was not to make changes to any of the existing code and only have truncation related tests in a separate file. Please revert all changes to pre-existing code, where trunctaion logic is not at all validated

Reply:
i moved those utests back into test_statushandler.py and i left them as it was no changes to variables and descriptions. done